engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -122,6 +122,9 @@ void runSemanticsTests() {
   group('tabs', () {
     _testTabs();
   });
+  group('dialogs', () {
```

Comment:
I'd either put this into the existing 'route' group (see comment about combining all routes into the same library), or put it next to the 'route' group since the two a closely related.